api/jobs.yaml

```diff
@@ -407,10 +407,18 @@ definitions:
     properties:
       results:
         type: array
+        required: true
         items:
           $ref: '#/definitions/QueryJobsResult'
+      totalKnownResults:
```

Comment:
I'd probably prefer `totalSize`, or if not that then at least `totalResults`. This API mostly follows the Google Cloud API style guide for pagination, which recommends `totalSize`:

https://cloud.google.com/apis/design/design_patterns#list_pagination